glean-core/rlb/tests/schema.rs

```diff
@@ -10,7 +10,7 @@ use serde_json::Value;
 
 //use glean::private::{DenominatorMetric, NumeratorMetric, RateMetric};
 use glean::net::UploadResult;
-use glean::{ClientInfoMetrics, CommonMetricData, Configuration};
+use glean::{ClientInfoMetrics, /*CommonMetricData,*/ Configuration};
```

Comment:
More comment lint

Reply:
Getting the linter to shut up. Really I should just leave it in until I removed the corresponding use (currently commented out)